desktop/src-tauri/src/main.rs

```diff
@@ -93,6 +93,7 @@ fn main() -> anyhow::Result<()> {
             resources_handles: Arc::new(Mutex::new(vec![])),
         })
         .plugin(logging::build_plugin())
+        // .plugin(tauri_plugin_deep_link::init())
```

Comment:
Is that wanted? That will basically disable the deep links doesn't it?

Reply:
oops, I should remove that. No i didn't end up updating that plugin as it didn't work. That was my attempt to use the new API that had a breaking change